.github/workflows/health-check.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '10.x', '12.x', '14.x' ]
+        node: [ '12.x', '14.x', '16.x', '18.x' ]
```

Comment:
as this is already breaking should we drop 12 and 14 ?

Reply:
just leaving it in for now